Mikellorza/liferay-portal
# LPS-179500 Add description to breadcrumb template setting

[Link to the story](https://issues.liferay.com/browse/LPS-179500)

Motivation
=======
Add description to breadcrumb template setting.

Proposed Solution
========
Create a description for the setting.

Steps to Verify:
----------------
 Go to System Settings > Breadcrumbs > Breadcrumb Template to see the description.
